daemon/snap.go

```diff
@@ -154,10 +155,10 @@ func allLocalSnapInfos(st *state.State, all bool) ([]aboutSnap, error) {
 
 // appJSON contains the json for snap.AppInfo
 type appJSON struct {
-	Name         string   `json:"name"`
-	Daemon       string   `json:"daemon"`
-	DesktopFiles []string `json:"desktop-files"`
-	Aliases      []string `json:"aliases,omitempty"`
+	Name        string   `json:"name"`
+	Daemon      string   `json:"daemon"`
+	DesktopFile string   `json:"desktop-files,omitempty"`
```

Comment:
Sorry, last sentence (now removed) was the original sentence which I intended to drop in favor of the actual explanation presented above.